.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea
```

Comment:
Best to leave this out. This should be done in a global .gitignore file.
https://murze.be/2014/12/create-a-global-gitignore/